app/code/Magento/InventoryInStorePickup/Model/IsOrderReadyForPickup.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model;
+
+use Magento\InventoryInStorePickup\Model\Order\IsFulfilled;
+use Magento\InventoryInStorePickupApi\Api\IsOrderReadyForPickupInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+
+class IsOrderReadyForPickup implements IsOrderReadyForPickupInterface
+{
+    /**
+     * @var \Magento\InventoryInStorePickup\Model\Order\IsFulfilled
```

Comment:
why do you use full names here? but not put everything into the header part?

Reply:
I put everything into the header part, BUT I strongly believe that class name should be full in phpdocs to preserve readability. Just compare: `Product` and  `\Magento\Catalog\Block\Adminhtml\Category\Tab\Product`, - the second case is more explicit.

Reply:
This is like discussing tabs vs spaces. Let's just remain consistent with the rest of the Inventory

Reply:
It is painful for me - but I have no choice anyway